frontend/src/components/BlogList.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from "react";
+import '../css/components/BlogList.scss';
+import BlogCard from "./BlogCard";
+import blog_img from "../images/blog_img.png";
+import avatar_image from "../images/avatar.png";
+import Avatar from "./Avatar.jsx";
+
+
+class BlogList extends Component {
+    render() {
+        return (
+
+            <div className="BlogList">
+
+                <div className="hero-section">
+                    <div className=" grid-container top">
+                        <div className=" grid-x grid-margin-x">
+
+                            <div className="cell large-12  ">
+                                <div className="BlogList_header">
+                                    <div className="BlogList__header__title">
+                                        <h1 >Stories</h1>
+                                    </div>
+                                    <div className=" grid-container top">
+                                        <div className=" grid-x grid-margin-x">
+
+                                            <div className="cell large-5 ">
+                                            <div BlogList__img>
+                                                <img src={blog_img} alt="blog_img" className="guidelines-card__image" />
+                                            </div>
+                                            </div>
+
+                                            <div className="cell large-7 ">
+                                                <div className="BlogList__header">
+                                                    <div className="BlogList__header__title">
```

Comment:
You can remove this `div` and directly give that class `BlogList__header__title` to h1. Also, make sure you remove the fixed width and height from the css.